doc/manual/source_localization/forward.rst

```diff
@@ -251,143 +251,6 @@ with positive direction up.
-
-The format of the text format surface files is the following:
-
-  | <*nvert*>
-  | <*vertex 1*>
-  | <*vertex 2*>
-  | ...
-  | <*vertex nvert*>
-  | <*ntri*>
-  | <*triangle 1*>
-  | <*triangle 2*>
-  | ...
-  | <*triangle ntri*> ,
-
-where <*nvert*> and <*ntri*> are
-the number of vertices and number of triangles in the tessellation,
-respectively.
-
-The format of a vertex entry is
-one of the following:
-
-**x y z**
-
-    The x, y, and z coordinates of the vertex location are given in
-    mm.
-
-**number x y z**
-
-    A running number and the x, y, and z coordinates are given. The running
-    number is not considered by mne_tri2fiff. The nodes must be thus
-    listed in the correct consecutive order.
-
-**x y z nx ny nz**
-
-    The x, y, and z coordinates as well as the approximate vertex normal direction
-    cosines are given.
-
-**number x y z nx ny nz**
-
-    A running number is given in addition to the vertex location and vertex
-    normal.
-
-Each triangle entry consists of the numbers of the vertices
-belonging to a triangle. The vertex numbering starts from one. The
-triangle list may also contain running numbers on each line describing
-a triangle.
-
-.. _BEHCBDDE:
-
-Topology checks
-===============
-
-If the ``--check`` option is specified, the following
-topology checks are performed:
-
-- The completeness of each surface is
-  confirmed by calculating the total solid angle subtended by all
-  triangles from a point inside the triangulation. The result should
-  be very close to :math:`4 \pi`. If the result
-  is :math:`-4 \pi` instead, it is conceivable
-  that the ordering of the triangle vertices is incorrect and the
-  ``--swap`` option should be specified.
-
-- The correct ordering of the surfaces is verified by checking
-  that the surfaces are inside each other as expected. This is accomplished
-  by checking that the sum solid angles subtended by triangles of
-  a surface :math:`S_k` at all vertices of another
-  surface :math:`S_p` which is supposed to be
-  inside it equals :math:`4 \pi`. Naturally, this
-  check is applied only if the model has more than one surface. Since
-  the surface relations are transitive, it is enough to check that
-  the outer skull surface is inside the skin surface and that the
-  inner skull surface is inside the outer skull one.
-
-- The extent of each of the triangulated volumes is checked.
-  If the extent is smaller than 50mm, an error is reported. This
-  may indicate that the vertex coordinates have been specified in
-  meters instead of millimeters.
-
-
-.. _CHDJFHEB:
-
-Computing the BEM geometry data
-###############################
-
-The utility :ref:`mne_prepare_bem_model` computes
-the geometry information for BEM. This utility is usually invoked
-by the convenience script :ref:`mne_setup_forward_model`,
-see :ref:`CIHDBFEG`. The command-line options are listed under
-:ref:`mne_prepare_bem_model`.
-
-
 .. _BJEIAEIE:
 
 Coil geometry information
```

Comment:
maybe somewhere in we should point to http://martinos.org/mne/stable/generated/mne.channels.fix_mag_coil_types.html#mne.channels.fix_mag_coil_types despite not being mandatory to use.